README.md

```diff
@@ -42,6 +42,23 @@ node = MongoDataNode(group='monolith')
 node.autorun()
 ```
 
+## Examples:
+
+### Base Server
+``` python
+from tyr.servers.server import Server
```

Comment:
i've never had to interact with it after and I doubt most people would. I like the simplicity of just the two lines. I feel that Developers would what's going on with that snipping and know that you could store it to a variable if needed.